### PR TITLE
fix(util): keeping the modal tabbing inside the focus area #90

### PR DESCRIPTION
**Pull request by jesussilverioea**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Fixes:** #91 

## Summary:

Currently, there is an issue when setting up the inert of the children when the component is inside another shadowDOM and the user starts tabbing and getting out of the focus zone.

## Type of change:

Please delete options that are not relevant.

- [ ] New capability
- [X] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [X] My update follows the CONTRIBUTING guidelines of this project
- [X] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: src/util.js [hunk 1]
```diff
@@ -15,6 +15,9 @@ export function makeSiblingsInert(node) {
    * @returns {void}
    */
   function handleInert(currentNode) {
+		if(!currentNode.parentNode && currentNode.host){
+			currentNode = currentNode.host;
+		}
```
Comment (blackfalcon) [inline src/util.js:20]: nit-pick

Not sure what happened to the tabbing here, but please correct the code formatting.

**Review by blackfalcon — CHANGES_REQUESTED** [review]

**Comment by blackfalcon** [comment]
Please adjust the commit message. See https://auro.alaskaair.com/contributing#:~:text=BREAKING%20CHANGE%20comment.-,Git%20commit%20messages,-Once%20you%20have for proper formatting.

File: src/util.js (hunk 1 above)
Comment (jesussilverioea) [reply src/util.js]: right formatting has been apply 👍

**Review by blackfalcon — APPROVED** [review]

**Pull request merged**